src/test/java/org/jabref/logic/exporter/XmpPdfExporterTest.java

```diff
@@ -143,6 +157,41 @@ void unsuccessfulExportToFileByPath(Path path) throws Exception {
         assertFalse(exporter.exportToFileByPath(databaseContext, filePreferences, path, abbreviationRepository));
     }
 
+    @Test
+    public void testRoundtripExportImport() {
+        Path originalFilePath = Path.of("original.pdf");
+        Path exportedFilePath = Path.of("exported.pdf");
+
+        try {
+            try (PDDocument document = new PDDocument()) {
+                PDPage page = new PDPage();
+                document.addPage(page);
+
+                try (PDPageContentStream contentStream = new PDPageContentStream(document, page)) {
+                    contentStream.beginText();
+                    contentStream.newLineAtOffset(25, 500);
+                    contentStream.setFont(new PDType1Font(Standard14Fonts.FontName.HELVETICA), 12);
+                    contentStream.showText("This PDF was created by JabRef. It demonstrates the embedding of BibTeX data in PDF files. Please open the file attachments view of your PDF viewer to see the attached files. Note that the normal usage is to embed the BibTeX data in an existing PDF.");
+                    contentStream.endText();
+                }
+                document.save(originalFilePath.toString());
+            }
+            exporter.exportToFileByPath(databaseContext, filePreferences, originalFilePath, abbreviationRepository);
+
+            Files.copy(Paths.get(originalFilePath.toString()), Paths.get(exportedFilePath.toString()), StandardCopyOption.REPLACE_EXISTING);
+
+            try (PDDocument importedDocument = Loader.loadPDF(new File(exportedFilePath.toString()))) {
+                assertNotNull(importedDocument, "The imported document should not be null");
```

Comment:
I would use JabRef's PDF Importer for this so you can compare the exported entries with the imported entries
PdfEmbeddedBibFileImporterTest

Reply:
I added the importer and commit it